DatadogRUM/Sources/Instrumentation/RUMInstrumentation.swift

```diff
@@ -78,22 +86,34 @@ internal final class RUMInstrumentation: RUMCommandPublisher {
             )
         }
 
-        if let threshold = longTaskThreshold, threshold > 0 {
-            longTasks = LongTaskObserver(threshold: threshold, dateProvider: dateProvider)
+        if let longTaskThreshold = longTaskThreshold {
+            if longTaskThreshold > Constants.minLongTaskThreshold {
+                longTasks = LongTaskObserver(threshold: longTaskThreshold, dateProvider: dateProvider)
+            } else {
+                DD.logger.error("`RUM.Configuration.longTaskThreshold` cannot be less than 0s. Long Tasks monitoring will be disabled.")
+            }
+        }
+
+        if let appHangThreshold = appHangThreshold {
+            if appHangThreshold >= Constants.minAppHangThreshold {
+                appHangs = AppHangsObserver(
+                    appHangThreshold: appHangThreshold,
+                    observedQueue: mainQueue,
+                    backtraceReporter: backtraceReporter,
+                    dateProvider: dateProvider,
+                    telemetry: telemetry
+                )
+            } else {
+                DD.logger.error("`RUM.Configuration.appHangThreshold` cannot be less than 0.1s. App Hangs monitoring will be disabled.")
+            }
```

Comment:
/question When the threshold is lower that the min, shouldn't we still enable the feature using the min value + print an warning? instead of not enable the instrumentation?